README.md

```diff
@@ -130,6 +130,10 @@ The exact viewport of the calendar. When these are specified, scrolling in the c
 
 **Note that you need to provide either `defaultTimeStart/End` or `visibleTimeStart/End` for the timeline to function**
 
+## utcOffset
+
+Define the offset you want to apply from UTC. Defaults value is your local timezone offset (e.g. moment().utcOffset())
```

Comment:
What input does `utcOffset` take?

Reply:
String or number.
[https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/09-utc-offset/](https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/09-utc-offset/)

Reply:
Cool 👌 perhaps we can add this as an example input to the `README.md`? What do you think?